tests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/project/ProjectNameValidationTest.java

```diff
@@ -38,61 +43,51 @@
 	public static final String PROJECT_NAME_MAX_LENGTH_ERROR = 
 			" Maximum length allowed is 63 characters for project name";
 	
+	@Before
+	public void setUp() {
+		openNewProjectShell();
+	}
+	
+	@After
+	public void cleanUp() {
+		closeNewProjectShell();
+	}
+	
 	@Test
 	public void testShortProjectName() {
-		openNewProjectShell();
 		new LabeledText(OpenShiftLabel.TextLabels.PROJECT_NAME).setText("s");
-		
-		try {
-			new DefaultText(PROJECT_NAME_SHORT_ERROR);
-		} catch (RedDeerException ex) {
-			fail("There is no validation message warning about small length of a project name.");
-		}
-		
-		closeNewProjectShell();
+		new DefaultText(PROJECT_NAME_SHORT_ERROR);
```

Comment:
I can only reiterate my objections: 
A test with an exception is a failing test. If this is not acceptable to you then we should make sure red deer behaves differently (ex. by not throwing an exception but by throwing an AssertionFailedError that your fail() is throwing). Repeating the very same catch and fail() in tests makes no sense. It's unnecessary noise imho.
I'd like to progress here, but it looks like we are stuck in arguments. @jeffmaury @alexeykazakov Further opinions please?